### ISSUE TITLE
it does not seems like localizer.withculture() work

### ISSUE THREAD

**Issue by bondo11** [AlexTeixeira/Askmethat-Aspnet-JsonLocalizer#65]
If i use 
```
@{
var desiredLanguageLocalizer = Localizer.WithCulture(new CultureInfo("da"));
}
<p>@desiredLanguageLocalizer['myKey']</p>
```

It does not seem to use the culture i selected from above, but the "default one"..

I need to be able to use a predefined language set in a db, and pretty much have to ignore what the user have it the users accept-language/cookie.. But it seems i cant do anything to make it work, without a pageload or redirect which seems silly?

The above should be the normal way to force the localizer to use a predetermined language, but it seems to ignore it.

(the above is tried directly in the view, but is ignored. I have not tried directly from the controller, but i guess its the same)

**Comment by AlexTeixeira** [comment]
Hi,

That's true, it was not working at all because I do not set the new culture.
It's fixed and I will release a new version today.

**Comment by AlexTeixeira** [comment]
Fixed on 2.2.0 preview 1. You can have a look
